Dockerfile

```diff
@@ -5,13 +5,25 @@ ENV BAMBOO_HOME     /var/atlassian/bamboo
 ENV BAMBOO_INSTALL  /opt/atlassian/bamboo
 ENV BAMBOO_VERSION  6.0.0
 
+# Copy in the gpg key for github_git-lfs so that we can load git-lfs
+# into the image.
+COPY github-lfs-repo.gpgkey /tmp
+
 # Install Atlassian Bamboo and helper tools and setup initial home
 # directory structure.
 RUN set -x \
+    && echo 'Adding repository and required key to install git-lfs' \
+    && apt-key add /tmp/github-lfs-repo.gpgkey \
+    && rm /tmp/github-lfs-repo.gpgkey \
+    && apt-get update --quiet \
+    && apt-get install --quiet --yes --no-install-recommends \
+               apt-transport-https \
+               xmlstarlet \
+    && echo 'deb http://packagecloud.io/github/git-lfs/debian/ jessie main' > /etc/apt/sources.list.d/github_git-lfs.list \
```

Comment:
If this is removed, then git-lfs cannot be (easily) installed. apt-transport-https is needed for the git-lfs repo, as it is only exported over https, via a redirect.